src/main/java/reposense/system/StreamGobbler.java

```diff
@@ -24,6 +24,11 @@ public String getValue() {
         return value;
     }
 
+    /**
+     * Reads from input stream {@code is} and stores it into {@code value}.
+     * This method will attempt to iteratively read 8KB of data from {@code is}, until no
```

Comment:
I think its not needed to state the second point `This method will attempt...`

Reply:
Sure, I have removed it. It does seem to be a bit redundant.